.github/workflows/recreate-qa-database.yml

```diff
@@ -0,0 +1,80 @@
+name: Replace Postgres Database
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Environment
+        default: qa
+        required: true
+
+env:
+ DOCKER_REPOSITORY: ghcr.io/dfe-digital/teaching-vacancies
+
+jobs:
+  turnstyle:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 20
+    steps:
+    - uses: DFE-Digital/github-actions/turnstyle@master
+      name: Check workflow concurrency
+      with:
+        initial-wait-seconds: 12
+        poll-interval-seconds: 20
+        same-branch-only: true
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  deploy-app:
+    name: Deploy master image tag to ${{ github.event.inputs.environment }}
+    runs-on: ubuntu-20.04
+    steps:
+    - uses: actions/checkout@v2
+      name: Checkout Code
+
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v1
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: eu-west-2
+        role-to-assume: Deployments
+        role-duration-seconds: 3600
+        role-skip-session-tagging: true
+
+    - name: Pin Terraform version
+      uses: hashicorp/setup-terraform@v1
+      with:
+        terraform_version: 0.15.5
+
+    - name: Set environment variables for non-review environments
+      run: |
+        PARAMETER_STORE_ENVIRONMENT=${{ github.event.inputs.environment }}
+        DEPLOY_ENV=${{ github.event.inputs.environment }}
+        echo "PARAMETER_STORE_ENVIRONMENT=${PARAMETER_STORE_ENVIRONMENT}" >> $GITHUB_ENV
+        echo "DEPLOY_ENV=${DEPLOY_ENV}" >> $GITHUB_ENV
```

Comment:
Is DEPLOY_ENV used anywhere?

Reply:
It would be if Idecided to say:-
make ${{env.DEPLOY_ENV }} ci terraform-app-database-replace CONFIRM_REPLACE=yes.............. and remove `qa`

Reply:
or remove it and use `make ${{ github.event.inputs.environment }} ......`